recipes/pyigrf/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pyIGRF" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ae9e27c6bcbb2730abd1069acd5d0ca72d8fa3d8e71ac75dc1115e52a1b196dc"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  imports:
+    - pyIGRF
+
+about:
+  home: "https://github.com/zzyztyy/pyIGRF"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "IGRF-12 Model by Python"
+  doc_url: ""
```

Comment:
If there is nothing to fill some fields with, you can just remove them. Please do that with `doc_url`.